mailpoet/tests/integration/Mailer/WordPress/WordpressMailerTest.php

```diff
@@ -43,9 +44,7 @@ public function testItFormatsTextNewsletterForMailer() {
         ],
       ]))
       ->willReturn(['response' => true]);
-    $fallbackMailer = $this->createMock(Mailer::class);
-    $fallbackMailer->expects($this->never())->method('send');
-    $mailerFactoryMock = $this->createMailerFactoryMock($mailer, $fallbackMailer);
```

Comment:
Every single test was checking that the fallback was not called when the primary mailer was successful. I kept the check only in one test case. I think it was redundant and the new approach require more code to prepare the mock for the fallback.